com.beeper.Beeper.metainfo.xml

```diff
@@ -0,0 +1,78 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.beeper.Beeper</id>
+  <name>Beeper</name>
+  <project_license>LicenseRef-proprietary</project_license>
+  <developer_name>Beeper Team</developer_name>
+  <summary>All your chats in one app. Yes, really.</summary>
+  <metadata_license>CC0-1.0</metadata_license>
+  <url type="homepage">https://www.beeper.com/</url>
+  <url type="help">https://help.beeper.com/</url>
+  <description>
+    <p>We're building the best chat app on Earth</p>
```

Comment:
Maybe revising the copy to be more honest given that 1. it's not open source 2. it doesn't run on non x86_64.

Reply:
Fair I was just copying what it said on their website :P